### PR TITLE
fix list styling

### PR DESCRIPTION
**Pull request by archiewood**
### Description

- Fixes list styling

### Checklist

- [x] For UI or styling changes, I have added a screenshot or gif showing before & after
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

## Before -> After

image

## Sub list styling

CleanShot 2023-04-21 at 10 30 40

### CODE REVIEW

File: .changeset/wet-eyes-buy.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'@evidence-dev/evidence': patch
+'@evidence-dev/preprocess': patch
+'@evidence-dev/components': patch
+---
```
Comment (archiewood) [inline .changeset/wet-eyes-buy.md:5]: @Winterhart changesets automatically added all of these packages, though I'm not sure all are needed.

I would have suspected just components?

Should I remove the others do you think?

**Review by hughess — COMMENTED** [review]
Tested on Mac Chrome and it looks great.

Safari has an issue with the ordered list for some reason:
CleanShot 2023-04-21 at 11 12 14@2x

**Comment by archiewood** [comment]
Looks great I think just needs formatting

**Comment by archiewood** [comment]
Looks great!

**Pull request merged**